src/main/java/net/mcreator/element/parts/gui/Image.java

```diff
@@ -74,10 +74,10 @@ public java.awt.Image getImage(Workspace workspace) {
 		int cw, ch;
 		if (this.use1Xscale) {
 			cw = actualImage.getWidth(null) / 2;
-			ch = actualImage.getHeight(null) / 2;
+			ch = animatedFrame != null ? cw : actualImage.getHeight(null) / 2;
 		} else {
 			cw = actualImage.getWidth(null);
-			ch = actualImage.getHeight(null);
+			ch = animatedFrame != null ? cw : actualImage.getHeight(null);
 		}
 		g.drawImage(actualImage, cx, cy, cw, ch, wysiwygEditor);

```

Comment:
Should only the first frame be rendered if the image is animated?

Reply:
Yes, this is ok approach

Reply:
No, it renders the entire image ATM.

Reply:
Well this is not ok as it shows wrong size. Make it show 1st frame only

Reply:
✔️ Fixed